src/storage/txn/scheduler.rs

```diff
@@ -710,12 +713,13 @@ impl Scheduler {
     fn release_lock(&mut self, lock: &Lock, cid: u64) {
         let wakeup_list = self.latches.release(lock, cid);
         for wcid in wakeup_list {
-            self.wakeup_cmd(wcid);
+            self.lock_and_get_snapshot(wcid);
         }
     }
 
-    /// Wakes up the next command for execution.
-    fn wakeup_cmd(&mut self, cid: u64) {
+    /// Tries to acquire all the necessary latches. If successful, initiates a get
+    /// snapshot for furthur processing.
```

Comment:
If successful, initiates a get snapshot for furthur processing. ->If all the necessary ltches are acquired, the method initiates a get snapshot operation for furthur processing.